### PR TITLE
Support Tahoma v1 without twilight_city

### LINKED ISSUE #652: Support Tahoma v1 without `twilight_city`
Configuring an Overkiz integration in Home Assistant Core 2022.10.5 (not a custom integration) with a Tahoma v1 box currently fails with the following error:

```
Logger: homeassistant.config_entries
Source: components/overkiz/__init__.py:61
First occurred: 22:08:48 (1 occurrences)
Last logged: 22:08:48

Error setting up entry me@mydomain.org for overkiz
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 365, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/overkiz/__init__.py", line 61, in async_setup_entry
    setup, scenarios = await asyncio.gather(
  File "/usr/local/lib/python3.10/site-packages/backoff/_async.py", line 133, in retry
    ret = await target(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/pyoverkiz/client.py", line 389, in get_setup
    setup = Setup(**humps.decamelize(response))
  File "/usr/local/lib/python3.10/site-packages/pyoverkiz/models.py", line 61, in __init__
    self.location = Location(**location) if location else None
TypeError: Location.__init__() missing 1 required keyword-only argument: 'twilight_city'
```

Restarting Home Assistant does not help.

### PR DESCRIPTION
**Pull request by pdecat**
This PR tentatively tries to resolve #652.

### CODE REVIEW

File: tests/fixtures/setup/setup_tahoma_1.json [hunk 1]
```diff
@@ -0,0 +1,7372 @@
...
+        "city": "* * *",
+        "country": "*",
+        "postalCode": "*",
+        "addressLine1": "*, * * *",
+        "addressLine2": "",
+        "timezone": "Europe/Paris",
+        "longitude": "*",
+        "latitude": "*",
+        "twilightMode": 2,
+        "twilightAngle": "CIVIL",
```
Comment (pdecat) [inline tests/fixtures/setup/setup_tahoma_1.json:17]: This payload was just copied from `setup_tahoma_3.json` with the only change being the removal of `twilightCity`.

Not sure yet how to properly capture an actual Tahoma v1 payload.

**Comment by pdecat** [comment]
Patching my live supervised Home Assistant container with this change seems to work-around the issue:

image

File: pyoverkiz/models.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ class Location:
     latitude: str = field(repr=obfuscate_string, default=None)
     twilight_mode: int
     twilight_angle: str
-    twilight_city: str
+    twilight_city: str = field(repr=obfuscate_string, default=None)
```
Comment (tetienne) [inline pyoverkiz/models.py:85]:
```suggestion
    twilight_city: str | None = None
```

File: pyoverkiz/models.py [hunk 3]
```diff
@@ -104,7 +104,7 @@ def __init__(
         latitude: str = field(repr=obfuscate_string, default=None),
         twilight_mode: int,
         twilight_angle: str,
-        twilight_city: str,
+        twilight_city: str = field(repr=obfuscate_string, default=None),
```
Comment (tetienne) [inline pyoverkiz/models.py:107]:
```suggestion
        twilight_city: str | None = None
```

File: tests/fixtures/setup/setup_tahoma_1.json (hunk 1 above)
Comment (tetienne) [reply tests/fixtures/setup/setup_tahoma_1.json:17]: Thx for adding test. We haven’t yet a fixture for Tahoma v1 indeed. You can use you own one:
 * Go to tahomalink.com 
 * Enable the Inspector
 * Open the network tab filtering on the word `setup`
 * Log in
 * You will see a request `setup?_=xxxx`
 * Copy the content of the response
 * Paste it within setup_tahoma_1.json
 * Finally run this [script](https://github.com/iMicknl/python-overkiz-api/blob/main/utils/mask_fixtures.py) to hide sensitive data.

File: pyoverkiz/models.py (hunk 2 above)
Comment (tetienne) [reply pyoverkiz/models.py:85]: IMO We don’t need to hide this data. In my case, it returns Paris while I live in the south of France.

**Review by tetienne — CHANGES_REQUESTED** [review]
Thank you for your fix. I’ve done some remarks.

File: tests/fixtures/setup/setup_tahoma_1.json (hunk 1 above)
Comment (pdecat) [reply tests/fixtures/setup/setup_tahoma_1.json:15]: Done!

**Comment by pdecat** [comment]
I've reverted the changes to other setup payload JSON files, and fixed the newly added one.

File: tests/fixtures/setup/setup_tahoma_1.json [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+                "creationTime": 1434967038000,
+                "lastUpdateTime": 1434975346000,
+                "label": "**",
+                "type": 44,
+                "oid": "2c9080963b51c2f5013c11754f744da7",
+                "subPlaces": []
+            },
+            {
+                "creationTime": 1434967038000,
+                "label": "**é**",
```
Comment (tetienne) [inline tests/fixtures/setup/setup_tahoma_1.json:128]: Funny we don't catch this

**Review by tetienne — APPROVED** [review]
Perfect!

**Pull request merged**